### DISCUSSION TITLE
3.21.24.0 Q and A

### DISCUSSION THREAD

**Discussion by Ongjming** [SmartlyDressedGames/Unturned-3.x-Community#2747] [General]
Hi, thanks so much for the update!
I just have one question with regards to this
>Experience modifier for consumeable items.

How does it work and how would a line in the .dat file look like?

Thanks in advance to anyone who answers! :D Cheers!

EDIT: I realised that there might be different questions and answers so I picked the "general discussion" category instead of the "Q & A".

**Comment by DiFFoZ** [comment]
Just line
 ```
Experience **int**
```

Examples:
`Experience 15` +15 exp

`Experience -15` -15 exp

**Reply by Ongjming** [reply]
Thanks so much!
